### PR TITLE
Added more columns to show

### PR DESCRIPTION
**Pull request by jshaofuturewei**

### CODE REVIEW

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 1]
```diff
@@ -179,7 +179,12 @@ func (p *Process) SendPodToCluster() {
 			klog.Warningf("Failed to get token from host %v", host)
 			return
 		}
+		pod.Status.DispatcherName = p.name
 		if pod.ObjectMeta.DeletionTimestamp != nil {
+			currentTime1 := time.Now().UTC()
+			klog.V(2).Infof("******** %v  Pod %s has been deleted", currentTime1.Format("2006-01-02 15:04:05.000000"), pod.ObjectMeta.Name)
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:185]: We can remove this line.

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 2]
```diff
@@ -193,44 +198,55 @@ func (p *Process) SendPodToCluster() {
...
+				if err == nil {
+					klog.V(3).Infof("Deleting request for pod %v has been sent to %v", pod.ObjectMeta.Name, host)
+				} else {
+					klog.Warningf("Failed to delete the pod %v with error %v", pod.ObjectMeta.Name, err)
+				}
+			}()
+
 		} else {
+			currentTime1 := time.Now().UTC()
+			klog.V(2).Infof("******** %v  Pod %s has been created", currentTime1.Format("2006-01-02 15:04:05.000000"), pod.ObjectMeta.Name)
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:212]: ditto

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 3]
```diff
@@ -193,44 +198,55 @@ func (p *Process) SendPodToCluster() {
...
+			klog.V(2).Infof("******** Pod %s has been created at %d", pod.ObjectMeta.Name, currentTime1.UnixNano()/int64(time.Millisecond))
+
 			p.totalPodCreateNum += 1
 			// Calculate create latency
 			podCreateTime := pod.CreationTimestamp
 			currentTime := time.Now().UTC()
 			duration := (currentTime.UnixNano() - podCreateTime.UnixNano()) / 1000000
 			p.totalCreateLatency += duration
 			createLatency := int(duration)
+			klog.V(2).Infof("******** %v  Pod %s has been scheduled", currentTime.Format("2006-01-02 15:04:05.000000"), pod.Name)
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:222]: Let's consolidate this with the previous one

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 4]
```diff
@@ -193,44 +198,55 @@ func (p *Process) SendPodToCluster() {
...
-				pod.Status.Phase = v1.PodFailed
-				if _, err := p.clientset.CoreV1().Pods(pod.ObjectMeta.Namespace).UpdateStatus(pod); err != nil {
-					klog.Warningf("Failed to create the pod %v with error %v", pod.ObjectMeta.Name, err)
+					pod.Status.Phase = v1.PodFailed
+					if _, err := p.clientset.CoreV1().Pods(pod.ObjectMeta.Namespace).UpdateStatus(pod); err != nil {
+						klog.Warningf("Failed to create the pod %v with error %v", pod.ObjectMeta.Name, err)
+					}
 				}
-			}
+			}()
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:249]: please add a msg to klog for openstack VM request sending failure

**Review by kimeunju108 — APPROVED** [review]
LGTM

**Review by CoderKevinZhang — APPROVED** [review]

File: globalscheduler/controllers/dispatcher/dispatcher_process.go [hunk 5]
```diff
@@ -213,24 +217,26 @@ func (p *Process) SendPodToCluster() {
...
+						klog.V(3).Infof("The pod %v has been updated its apiserver database status successfully with the instance id %v", updatedPod, instanceId)
 
+					} else {
+						klog.Warningf("The pod %v failed to update its apiserver database status  with the error %v", pod.ObjectMeta.Name, err)
+					}
 				} else {
-					klog.Warningf("Failed to update the pod %v with error %v", pod.ObjectMeta.Name, err)
+					pod.Status.Phase = v1.PodFailed
+					if _, err := p.clientset.CoreV1().Pods(pod.ObjectMeta.Namespace).UpdateStatus(pod); err != nil {
+						klog.Warningf("The openstack vm for the  pod %v failed to create with the error %v", pod.ObjectMeta.Name, err)
```
Comment (cathyhongzhang) [inline globalscheduler/controllers/dispatcher/dispatcher_process.go:236]: The klog here should log the failure of updating the API server DB. We need to add another klog line outside the "if" to log the msg for openstack VM creation sending failure

**Review by cathyhongzhang — APPROVED** [review]
LGTM

**Pull request merged**